includes/class-wc-order-item-product.php

```diff
@@ -167,7 +167,7 @@ public function set_taxes( $raw_tax_data ) {
 	 */
 	public function set_variation( $data ) {
 		foreach ( $data as $key => $value ) {
-			$this->add_meta_data( str_replace( 'attribute_', '', $key ), $value, true );
```

Comment:
This change might be an issue for backwards compatibility. I believe some code uses that prefix to identify attribute rows. Why was this removed can I ask?